multisig2/src/lib.rs

```diff
@@ -127,11 +128,12 @@ impl MultiSigContract {
     /// @params num_confirmations: k of n signatures required to perform operations.
     #[init]
     pub fn new(members: Vec<MultisigMember>, num_confirmations: u32) -> Self {
-        assert!(!env::state_exists(), "Already initialized");
-        assert!(
```

Comment:
it's easier to replace this with to keep the same number of instructions but keep the code pretty much the same.
```
#[inline]
fn assert(condition: bool, error: &str) {
    if condition {
        env::panic_str(error);
    }
}
```